src/common/BaseAnnotator.ts

```diff
@@ -127,6 +127,10 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch(store.setRotationAction(rotation));
         this.store.dispatch(store.setScaleAction(scale));
         this.store.dispatch(store.setIsInitialized());
+
+        // Set discoverability feature in redux
+        const isDiscoverabilityFeatureEnabled = this.isFeatureEnabled('discoverability');
```

Comment:
It might make more sense to actually load the whole features object into the initial state of the store: https://github.com/box/box-annotations/blob/master/src/common/BaseAnnotator.ts#L66. This probably won't be changed at runtime so we could get rid of the set action and reducer code

Reply:
Good catch - will address.